README.md

```diff
@@ -52,20 +47,14 @@ We recommend [direnv](https://direnv.net/) for managing your environment variabl
 
 - **Returns**: An array of `Report` objects.
 - **Purpose**: To provide impact reports to the UI.
-- **Implementation Details**: Uses `fetchReports()` from `server/impactReportHelpers.ts`.
+- **Implementation Details**: Uses `fetchReports()` from `lib/impact-reports.ts`.
 
 ### Server Functions
 
 Located in `app/server/impactReportHelpers.ts`:
 
 - `fetchReports`: Function to retrieve reports, including interaction with Hypercerts.
 
-### Data Models
-
-- **Impact Report**: The report or stories that have been published previously and verified to actually produce an impact.
-- **Hypercert**: A token representing a claim of impactful work, which is fractionable and transferable, conforming to the ERC-1155 standard for semi-fungible tokens.
-- **Hypercert Metadata**: A set of data associated with a Hypercert, detailing the scope of work, contributors, impact, and rights, stored on IPFS.
-
 ### Separation of Concerns
```

Comment:
might be getting too deep with this

Reply:
ok so you want to remove
- both references to React
- entire "Data Models" section
- all of "Server Design" section, or leave as it but no additional content?
- keep the expanded content in "Helpful References"

correct or no?

Reply:
yeah, you can remove them imo. the deeper we get the more we'll have to change if the tech changes